source/_integrations/sun.markdown

```diff
@@ -51,23 +51,26 @@ sun:
 <img src='/images/screenshots/more-info-dialog-sun.png' />
 </p>
 
-## Implementation Details
+## Automation trigger
 
 The sun's event listener will call the service when the sun rises or sets with
 an offset.
 
-The sun event need to have the type 'sun', which service to call,
-which event (sunset or sunrise) and the offset.
+The sun trigger need to have the type 'sun', which event (sunset or sunrise) and an optional offset.
 
 ```json
 {
-    "type": "sun",
-    "service": "switch.turn_on",
+    "platform": "sun",
     "event": "sunset",
     "offset": "-01:00:00"
 }
 ```
 
+| Key name | Description |
+| --------- | ----------- |
+| `event` | Possible values  `sunset`,  `sunrise `
+| `offset` | An optional offset specified in HH:MM:SS from event trigger. <br> Positive after event is triggered. <br> Negative before event is triggered
```

Comment:
- I think we should avoid "event trigger". Events in Home Assistant are a different thing; let's add context to it by saying sun trigger.

- Postive negative adds/subtracts from the time. I don't think we need to clarify that (besides, the current suggested wording was very unclear, IMHO).

```suggestion
| `offset` | An optional offset for the sun event trigger, in a positive or negative number of seconds, or specified in `HH:MM:SS` or `-HH:MM:SS`.
```

Reply:
It was actually positive/negative values that I struggled with, which caused me to create this PR.